packages/eas-cli/src/commands/env/exec.ts

```diff
@@ -111,7 +118,8 @@ export default class EnvExec extends EasCommand {
     environmentVariables: Record<string, string>;
   }): Promise<void> {
     Log.log(`Running command: ${chalk.bold(command)}`);
-    const spawnPromise = spawnAsync('bash', ['-c', command], {
+    const spawnPromise = spawnAsync(command, [], {
+      shell: true,
```

Comment:
I'm not sure if I fully understand why `bash -c` doesn't work on CI but it does on my local machine 🤔 Do you fully get it? Do you use Windows on CI by any chance?

Reply:
> Do you fully get it?

Hey Szymon, unfortunately, no... I am baffled too...

> Do you use Windows on CI by any chance?

Nope, I use Github actions with `runs-on: ubuntu-latest`.

---

It's totally weird, but even the quote issue seems to be happening only on CI...